hedera-node/src/test/java/com/hedera/services/store/contracts/precompile/HTSTestsUtil.java

```diff
@@ -37,6 +37,7 @@
 import org.apache.tuweni.units.bigints.UInt256;
 import org.hyperledger.besu.datatypes.Address;
 
+import java.math.BigInteger;
```

Comment:
I think we can remove all `BigInteger` related changes from the PR since we changed the approach